woke.txt

```diff
@@ -21,3 +21,11 @@ pronoun(s)
 meow
 nya+
 nonbinary
+mrrp
+>.<
```

Comment:
Replace `.` with `\.` because this is regex

Reply:
legit while adding them i thought about doing it but i wasn't sure if i had to do it, gonna fix